src/app/book/register/page.tsx

```
@@ -28,10 +30,78 @@
   const statusSelect = useSingleSelect<ReadingStatus>('unread');
   const genresSelect = useMultiSelect<string>();
 
+  const [selectedFile, setSelectedFile] = useState<File | null>(null);
```

ESLint

[error] 'selectedFile' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)